client/securedrop_client/gui/conversation/export/whistleflow_dialog.py

```diff
@@ -13,19 +14,18 @@
 from securedrop_client.export import ExportError, ExportStatus
```

Comment:
```suggestion
from securedrop_client.export import Export, ExportError, ExportStatus
```

Reply:
hmm there was a problem with this line - import was from the wrong file, which I've fixed. `Export` is imported a few lines below, in a slightly weird style with .... but that's the same way as it's done in the print dialog

Reply:
Ah ok, let's leave it the way it is then. Maybe it fixes a circular import or something?